.github/workflows/continuous-delivery.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
               { "release": "patch", "type": "ci" },
               { "release": "patch", "type": "improvement" },
               { "release": "patch", "type": "refactor" }
+              { "release": false, "scope": "skip-release" },
```

Comment:
How this scope is handled by action?

Reply:
@oleg-koval https://github.com/ridedott/release-me-action/pull/233#issuecomment-652842071